src/Microsoft.VisualStudio.ProjectSystem.Managed/Microsoft.VisualStudio.ProjectSystem.Managed.csproj

```diff
@@ -88,6 +88,9 @@
     <Compile Include="ProjectSystem\Rules\AnalyzerReference.xaml.cs">
       <DependentUpon>AnalyzerReference.xaml</DependentUpon>
     </Compile>
+    <Compile Include="ProjectSystem\Rules\ProjectDebugger.xaml.cs">
```

Comment:
Or is this a file that we used to have but don't need anymore after the move to launch profiles? If so, we should delete it. Either way, can you check it's setupauthoring as well?